core/src/main/java/org/elasticsearch/common/xcontent/ObjectParser.java

```diff
@@ -126,6 +126,7 @@ public ObjectParser(String name, boolean ignoreUnknownFields, @Nullable Supplier
      * @return a new value instance drawn from the provided value supplier on {@link #ObjectParser(String, Supplier)}
      * @throws IOException if an IOException occurs.
      */
+    @Override
```

Comment:
I wish we could enforce this with checkstyle. But it doesn't support it because it doesn't look at the bytecode. I *love* having these.